plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -162,7 +164,8 @@ void startDevServer(List<File> runnables, MessageConsoleStream stream) {
    * @param stream the stream to send development server process output to
    * @param debugPort the port to attach a debugger to if launch is in debug mode
    */
-  void startDebugDevServer(List<File> runnables, MessageConsoleStream stream, int debugPort) {
+  void startDebugDevServer(List<File> runnables, MessageConsoleStream stream,
```

Comment:
suggest stream --> console since this is not either of the things Java devs usually think of as a "stream"